.github/workflows/daily-nightly-jobs.yml

```diff
@@ -26,20 +29,26 @@ jobs:
         with:
           go-version: 1.18
 
-      - name: teardown minikube and docker
+      - name: teardown minikube, docker and kubectl
+        if: always()
         run: |
-          uptime
```

Comment:
uptime would be needed for cron

Reply:
I had temporary changes for testing, I have reverted